internal/codegen/tpl_stencil.go

```diff
@@ -119,23 +120,43 @@ func (s *TplStencil) Args() map[string]interface{} {
 //
 //	{{ stencil.ReadFile "myfile.txt" }}
 func (s *TplStencil) ReadFile(name string) (string, error) {
-	cwd, err := os.Getwd()
+	f, ok := s.exists(name)
+	if !ok {
+		return "", errors.Errorf("file %q does not exist", name)
+	}
+
+	b, err := io.ReadAll(f)
 	if err != nil {
 		return "", err
 	}
 
-	f, err := osfs.New(cwd).Open(name)
+	return string(b), nil
+}
+
+// Exists returns true if the file exists in the current directory
+//
+//	{{- if stencil.Exists "myfile.txt" }}
+//	{{ stencil.ReadFile "myfile.txt" }}
+//	{{- end }}
+func (s *TplStencil) Exists(name string) bool {
+	f, ok := s.exists(name)
```

Comment:
I didn't see much value in adding a test for this because it's using a function already covered (Readfile -> exists). I can add one if someone wants though